staging/src/k8s.io/client-go/util/certificate/certificate_manager.go

```diff
@@ -91,7 +94,7 @@ type Config struct {
 	// CertificateStore is a persistent store where the current cert/key is
 	// kept and future cert/key pairs will be persisted after they are
 	// generated.
-	CertificateStore Store
+	CertificateStore FileStore
```

Comment:
why switch to FileStore?

Reply:
in https://github.com/kubernetes/kubernetes/pull/80169/files#diff-75a52f1ad830d3c146427372bebebf91R433 need use CurrentPath() to get "kubelet-client-current.pem" path, and this funcation is in FIleStore interface.